spec/Modules.tex

```diff
@@ -333,7 +357,8 @@ \subsection{Module Initialization}
 
 Module initialization occurs at program start-up.  All top-level
 statements in a module other than function and type declarations are
-executed during module initialization.
+executed during module initialization. Top-level modules and submodules
+that are not referred to will not be initialized.
```

Comment:
"that are not referred to" has ambiguous applicability. How about:

`Modules that are not referred to, including both top-level modules and submodules, will not be initialized.`